test/conformance/ingress/util.go

```diff
@@ -362,6 +374,11 @@ func RuntimeRequest(t *testing.T, client *http.Client, url string) *types.Runtim
 	}
 
 	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		t.Errorf("Unable to read response body: %v", err)
+		DumpResponse(t, resp)
```

Comment:
If we failed to read the body, DumpResponse probably will too, though I've yet to see a test to fail because of that 😊 .